### PR TITLE
[288] fix for level 1 moderator responses showing in moderation queue

### PR DESCRIPTION
**Pull request by richbrennan**
Fix for level 1 moderator responses showing in moderation queue https://github.com/mozilla/m-response/issues/288

### CODE REVIEW

File: mresponse/responses/query.py [hunk 1]
```diff
@@ -34,7 +34,9 @@ def not_approved(self):
         return self.filter(approved=False)
 
     def moderator_queue(self):
-        return self.not_approved()
+        return self.not_approved().exclude(
+            author__user_permissions__codename='can_bypass_community_moderation'
```
Comment (tm-kn) [inline mresponse/responses/query.py]: But we already exclude approved responses... And we did set that flag already.

https://github.com/mozilla/m-response/blob/6d31d3981caf676d96fedf08427e0b8bcc3e4e94/mresponse/responses/api/serializers.py#L41-L42

**Comment by richbrennan** [comment]
Hi @tm-kn 

I haven't been able to recreate the issue this time (using the same data as last time I tried...).

I wrote some tests to help with my investigation and have included in this PR, do they look like they've covered the likely scenarios where the problem would occur?

Thanks,

Rich.

File: mresponse/responses/tests.py [hunk 2]
```diff
@@ -68,3 +73,50 @@ def test_can_bypass_staff_moderation(self):
...
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(json.loads(response.content.decode())['text'], 'test')
+
+    def test_returning_no_responses_to_moderate_for_approved_response(self):
+        self.client.force_authenticate(user=UserFactory())
+        ResponseSerializerFactory.create(author=BypassCommunityModerationUserFactory())
+        response = self.client.get('/api/response/')
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(
+            json.loads(response.content.decode())['detail'],
```
Comment (tm-kn) [inline mresponse/responses/tests.py]: `response.json()['detail']`?

**Comment by richbrennan** [comment]
Thanks @tm-kn , have updated. Do you think the tests look like they've covered the likely scenarios where the problem would occur?

**Comment by tm-kn** [comment]
They seem good to me, yea.

**Pull request closed without merging**